examples/mnist/mnist/main.py

```diff
@@ -7,7 +7,6 @@
 import numpy as np
 
 
-# example of loading the mnist dataset and building baseline cnn model
 def preprocess(df: pd.DataFrame) -> Tuple[np.ndarray, np.ndarray]:
     data_X = df.drop('label', axis=1).to_numpy()
     data_X = np.reshape(data_X, (len(data_X), 28, 28, 1)) / 255.  # normalize to range 0-1
```

Comment:
I recommend having one action in one line to improve readability
```suggestion
    data_X = np.reshape(data_X, (len(data_X), 28, 28, 1)) 
    data_X = data_X/ 255.  # normalize to range 0-1
```